api/peripheral_api/util/LedSign.js

```diff
@@ -0,0 +1,20 @@
+async function updateSignAxios(axios, req) {
```

Comment:
send in ...req.body as a param to updateSign

```suggestion
async function updateSign(data) {
```